includes/phonebook-functions.php

```diff
@@ -323,25 +323,34 @@ function format_phonebook_result_primary( $result, $is_dept, $is_org, $is_group
 function format_phonebook_result_location( $result, $is_dept, $is_org, $is_group ) {
 	ob_start();
 ?>
-	<?php if ( ! $is_group && $result->email ) : ?>
-		<span class="email d-block">
-			<a href="mailto:<?php echo $result->email; ?>"><?php echo $result->email; ?></a>
-		</span>
-	<?php endif; ?>
 	<?php if ( $result->building ) : ?>
-		<span class="location d-block">
-			<a href="https://map.ucf.edu/?show=<?php echo $result->bldg_id; ?>">
-				<?php echo phonebook_fix_name_case( $result->building ); ?>
-				<?php if ( $result->room ) : ?>
-					<?php echo ' - ' . $result->room; ?>
-				<?php endif; ?>
-			</a>
-		</span>
+		<div class="row mb-2">
+			<div class="col-3 col-md-12">
+				<span class="result-label text-default-aw text-uppercase" id="location-label">Location</span>
```

Comment:
`location-label` ID could be repeated in phonebook results and be invalid.  Need to make sure the ID value here is always unique, or change the `aria-labelledby` on the location link to just use `aria-label="Location"` instead